### PR TITLE
chore(download): add new optimize rleases to download page

### PR DESCRIPTION
**Pull request by misiekhardcore**

### CODE REVIEW

File: enterprise/content/download.md [hunk 1]
```diff
@@ -3819,13 +3819,13 @@ To install the demo distribution, please download one of the following archives:
   </thead>
   <tbody>
     <tr class="well">
-      <td>3.13.1</td>
+      <td>3.13.2</td>
       <td>7.19+, 7.20+. 7.21+</td>
       <td>8.9.0+</td>
-      <td>09.04.2024</td>
+      <td>13.05.2024</td>
```
Comment (RomanJRW) [inline enterprise/content/download.md:3825]: I think for patches we don't update the release dates

File: enterprise/content/download.md [hunk 2]
```diff
@@ -3921,13 +3921,13 @@ Please note that Optimize alpha releases are not suitable for production environ
   </thead>
   <tbody>
     <tr class="well">
-        <td>3.14.0-alpha1</td>
+        <td>3.14.0-alpha2</td>
         <td>7.19+, 7.20+, 7.21.0+</td>
         <td>8.9.0+</td>
         <td>14.05.2024</td>
```
Comment (RomanJRW) [inline enterprise/content/download.md:3927]: this is the only date that _does_ need updating 🙃

File: enterprise/content/download.md (hunk 1 above)
Comment (misiekhardcore) [reply enterprise/content/download.md:3825]: then this date 09.04.2024 is also wrong?

Comment (RomanJRW) [reply enterprise/content/download.md:3825]: No I think the date is correct, it should be the date that 3.13.0 came out. Then for patches it doesn't get updated (I might be wrong about this)

File: enterprise/content/download.md [hunk 3]
```diff
@@ -3855,43 +3855,43 @@ To get the production distribution, please download one of the following archive
   </thead>
   <tbody>
     <tr class="well">
-      <td>3.13.1</td>
+      <td>3.13.2</td>
       <td>7.19+, 7.20+, 7.21+</td>
       <td>8.9.0+</td>
-      <td>09.04.2024</td>
+      <td>13.05.2024</td>
```
Comment (RomanJRW) [inline enterprise/content/download.md:3861]: my comment above is the same here

File: enterprise/content/download.md [hunk 4]
```diff
@@ -3855,43 +3855,43 @@ To get the production distribution, please download one of the following archive
...
+        <a class="btn btn-sm btn-default optimize-download" href="https://downloads.camunda.cloud/enterprise-release/optimize/3.12.5/camunda-optimize-3.12.5-production.zip">zip</a>&nbsp;
+        <a class="btn btn-sm btn-default optimize-download" href="https://downloads.camunda.cloud/enterprise-release/optimize/3.12.5/camunda-optimize-3.12.5-production.tar.gz">tar.gz</a>
       </td>
     </tr>
     <tr class="well">
-      <td>3.11.9</td>
+      <td>3.11.10</td>
       <td>7.18+, 7.19+, 7.20+</td> 
       <td>8.8.0+</td>
       <td>09.04.2024</td>
```
Comment (RomanJRW) [inline enterprise/content/download.md:3881]: but yeah I see what you mean with regards to the dates being wrong here too 🙃 

Ok, let's just go with your original idea of updating them everywhere from now 👍

File: enterprise/content/download.md [hunk 5]
```diff
@@ -3921,13 +3921,13 @@ Please note that Optimize alpha releases are not suitable for production environ
   </thead>
   <tbody>
     <tr class="well">
-        <td>3.14.0-alpha1</td>
+        <td>3.14.0-alpha2</td>
         <td>7.19+, 7.20+, 7.21.0+</td>
         <td>8.9.0+</td>
-        <td>14.05.2024</td>
+        <td>13.05.2024</td>
```
Comment (RomanJRW) [inline enterprise/content/download.md:3927]: should this be `11.06.2024`?

**Review by RomanJRW — CHANGES_REQUESTED** [review]

**Review by RomanJRW — APPROVED** [review]

**Pull request merged**